tests/rol/vector_adaptor_02.cc

```diff
@@ -0,0 +1,141 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1998 - 2017 by the deal.II authors
```

Comment:
I can never remember how much we care about the dates, but perhaps you could ease my mind and adjust this one.

Reply:
Should it be something like
> Copyright (C) 2017 by the deal.II authors

Reply:
yes